### PR TITLE
MWPW-143708: Create Kodiak config.yaml

### PR DESCRIPTION
**Pull request by thedoc31**
Creates file required for MWPW-143708 and MWPW-143712

Resolves: 
- [MWPW-143708](https://jira.corp.adobe.com/browse/MWPW-143708)
- [MWPW-143712](https://jira.corp.adobe.com/browse/MWPW-143712)

**Test URLs:**
- Before: https://main--bacom--adobecom.hlx.live/?martech=off
- After: https://thedoc31-patch-1--bacom--adobecom.hlx.live/?martech=off

### CODE REVIEW

File: .kodiak/config.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+notifications:
+  jira:
+    default:
+      project: MWPW # Mandatory
+      filters:
+        include:
+          risk_rating: R5
+      fields:
+        assignee:
+          name: slavin
```
Comment (meganthecoder) [inline .kodiak/config.yaml:15]: @JasonHowellSlavin Should the tickets be assigned to Malik instead?

File: .kodiak/config.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - id: 546046 # Milo Bacom (business.adobe.com)
+
+notifications:
+  jira:
+    default:
+      project: MWPW # Mandatory
+      filters:
+        include:
+          risk_rating: R5
+      fields:
```
Comment (meganthecoder) [inline .kodiak/config.yaml:13]: Is it possible to set the Team to Magma?

Comment (thedoc31) [reply .kodiak/config.yaml:13]: @meganthecoder Possibly. I think Jed was able to do it for `caas`. There's more documentation on the available fields here if you are interested.  https://git.corp.adobe.com/kodiak-config-examples/default-notification-config/blob/main/Example%20configs/base-config.yaml

Comment (thedoc31) [reply .kodiak/config.yaml:13]: Found it. Added it in the next commit. Let me know if there's any other tweaks you'd like me to make.

File: .kodiak/config.yaml (hunk 1 above)
Comment (JasonHowellSlavin) [reply .kodiak/config.yaml:15]: Let's discuss at standup.

**Review by meganthecoder — APPROVED** [review]

**Review by Brandon32 — APPROVED** [review]

**Pull request merged**